core/src/main/java/org/apache/calcite/rel/externalize/RelJson.java

```diff
@@ -678,7 +678,8 @@ SqlOperator toOp(Map<String, Object> map) {
     if (class_ != null) {
       return AvaticaUtils.instantiatePlugin(SqlOperator.class, class_);
     }
-    return null;
+    throw new RuntimeException("No operator for " + name + " with kind: "
+        + kind + ", and syntax: " + syntax);
```

Comment:
Will it better to move this exception in `CalciteResource.java`?

Reply:
yes, I'll update

Reply:
Is this a breaking change ? Do you know the background that it returns null before ?

Reply:
Agree with @danny0405.

Reply:
Yes, this breaks a little. I don't know the background why it returns null before.
Hopes someone can help to give the background, but I doubt there exists some special reason, because null will always cause NPE somewhere else.